Libraries/library.ps1

```diff
@@ -0,0 +1,108 @@
+﻿$source = @"
+using System.Management.Automation;
+
+namespace ReportingServicesTools
+{
+    /// <summary>
+    /// Static class containing connection information to the reporting server.
+    /// </summary>
+    public static class ConnectionHost
+    {
+        /// <summary>
+        /// The name of the computer to connect to using WMI
+        /// </summary>
+        public static string ComputerName = "localhost";
+
+        /// <summary>
+        /// The name of the Database instance of the Report Server
+        /// </summary>
+        public static string Instance = "MSSQLSERVER";
+
+        /// <summary>
+        /// The version of the Report Server
+        /// </summary>
+        public static SqlServerVersion Version = SqlServerVersion.SQLServer2016;
+
+        /// <summary>
+        /// The credentials to use when connecting to the reporting services.
+        /// </summary>
+        public static PSCredential Credential;
```

Comment:
>Credential [](start = 35, length = 10)

What is the default value of this? I have seen you reference this in several places.